### PR TITLE
Docs/telemetry

### PR DESCRIPTION
**Pull request by arnoldna**
# Overview/Summary

Updated specification to clarify the placement of AVM Telemetry in Bicep modules.

## As part of this Pull Request I have

- [X] Read the Contribution Guide and ensured this PR is compliant with the guide
- [X] Checked for duplicate [Pull Requests](https://github.com/Azure/Azure-Verified-Modules/pulls)
- [ ] Associated it with relevant [GitHub Issues](https://github.com/Azure/Azure-Verified-Modules/issues) or ADO Work Items (Internal Only)
- [X] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Azure-Verified-Modules/)
- [X] Ensured PR tests are passing
- [X] Updated relevant and associated documentation (e.g. Contribution Guide, Docs etc.)

### CODE REVIEW

File: docs/content/specs/bicep/_index.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ To meet the requirements of [BCPFR2](/Azure-Verified-Modules/specs/bicep/#id-bcp
 
 #### ID: BCPFR4 - Category: Composition - Telemetry Enablement
 
-To meet the requirements of [SFR3](/Azure-Verified-Modules/specs/shared/#id-sfr3---category-telemetry---deploymentusage-telemetry) & [SFR4](/Azure-Verified-Modules/specs/shared/#id-sfr4---category-telemetry---telemetry-enablement-flexibility) you **MUST** use the below code sample in your AVM Modules to achieve this.
+To comply with specifications outlined in [SFR3](/Azure-Verified-Modules/specs/shared/#id-sfr3---category-telemetry---deploymentusage-telemetry) & [SFR4](/Azure-Verified-Modules/specs/shared/#id-sfr4---category-telemetry---telemetry-enablement-flexibility) you **MUST** incorporate the following code snippet into your Azure Verified Modules (AVM) modules. Place this code sample in the “top level” main.bicep file; it is not necessary to include it in any nested Bicep files.
```
Comment (matebarabas) [inline docs/content/specs/bicep/_index.md:88]:
```suggestion
To comply with specifications outlined in [SFR3](/Azure-Verified-Modules/specs/shared/#id-sfr3---category-telemetry---deploymentusage-telemetry) & [SFR4](/Azure-Verified-Modules/specs/shared/#id-sfr4---category-telemetry---telemetry-enablement-flexibility) you **MUST** incorporate the following code snippet into your modules. Place this code sample in the "top level" `main.bicep` file; it is not necessary to include it in any nested Bicep files (child modules).
```

Comment (matebarabas) [reply docs/content/specs/bicep/_index.md:88]: What about Terraform? We should provide the same guidance there too.

File: docs/content/specs/shared/_index.md [hunk 2]
```diff
@@ -111,7 +111,8 @@ These will also be provided as a comment on the module proposal, once accepted,
 
 {{< /hint >}}
 
-Modules **MUST** provide the capability to collect deployment/usage telemetry, via a blank ARM deployment, as detailed in [Telemetry](/Azure-Verified-Modules/help-support/telemetry/) further.
+Modules **MUST** provide the capability to collect deployment/usage telemetry, via a blank ARM deployment, as detailed in [Telemetry](/Azure-Verified-Modules/help-support/telemetry/) further. Telemetry data should be collected on the "top level" `main.bicep` or `main.telemetry.tf` file; it is not necessary to include it in any nested files (child modules).
```
Comment (matebarabas) [inline docs/content/specs/shared/_index.md:114]: @matt-FFFFFF, @mbilalamjad, @segraef - this extra sentence was added for clarity (see the parallel change for the Bicep documentation in the same PR), as there was confusion about whether the telemetryId should be added to the child modules too (if any). Please review/comment/approve as needed. Thanks!

Comment (segraef) [reply docs/content/specs/shared/_index.md:114]: lgtm, waiting for @matt-FFFFFF

**Review by segraef — APPROVED** [review]

File: docs/content/specs/shared/_index.md (hunk 2 above)
Comment (matt-FFFFFF) [reply docs/content/specs/shared/_index.md:114]: Also LGTM

**Review by matebarabas — APPROVED** [review]

**Pull request merged**